README.md

```diff
@@ -1 +1,48 @@
-# Portfolio_1
\ No newline at end of file
+# Portfolio: setup and mobile version skeleton
+
+![Alt text](images/portfolio.png?raw=true "Optional Title")
+
+## Learning objectives
+- Understand how to parse a Figma design to create a UI.
+- Flexbox to place elements in the page.
+- Build a personal portfolio site.
+- Use images and backgrounds to enhance the look of the website.
+=======
+
+## Built With
+
+- HTML/CSS
+- Node.JS
+
+## Getting Started
+
+If you want a copy of this file go to the github repository and download it
+
+`git clone https://github.com/Metsanpeitto/Portfolio_1`
+
+
```

Comment:
- [ ] Also mention after clone which file needs to run. In this case `index.html`.